src/jobs/models/jobManager.js

```diff
@@ -28,14 +24,16 @@ module.exports.reloadCronJobs = async function () {
 };
 
 module.exports.createJob = async function (job) {
+    const configData = await configHandler.getConfig();
+    const jobConnector = require(`./${configData.job_platform.toLowerCase()}/jobConnector`);
     let jobId = uuid.v4();
 
     try {
         await databaseConnector.insertJob(jobId, job);
         logger.info('Job saved successfully to database');
         let latestDockerImage = await dockerHubConnector.getMostRecentRunnerTag();
         let runId = Date.now();
-        let jobSpecificPlatformRequest = createJobRequest(jobId, runId, job, latestDockerImage);
+        let jobSpecificPlatformRequest = await createJobRequest(jobId, runId, job, latestDockerImage);
         if (job.run_immediately) {
```

Comment:
again, you do twice "get config", move the config as argument to createJobRequest - and createJobRequest will not be async anymore.

Reply:
Threre is only one call :   
let jobSpecificPlatformRequest = await createJobRequest(jobId, runId, job, latestDockerImage);
the red one is deleted row.

Reply:
👍